addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -51,66 +56,67 @@ options.registry.mailing_list_subscribe = mass_mailing_common.extend({
 options.registry.newsletter_popup = mass_mailing_common.extend({
     popup_template_id: "editor_new_mailing_list_subscribe_popup",
     popup_title: _t("Add a Newsletter Subscribe Popup"),
-    select_mailing_list: function (previewMode, value) {
+    /**
+     * @override
+     */
+    start: function () {
         var self = this;
-        return this._super(previewMode, value).then(function (mailing_list_id) {
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'read',
-                args: [[parseInt(mailing_list_id)], ['popup_content'], self.options.recordInfo.context],
-            }).then(function (data) {
-                self.$target.find(".o_popup_content_dev").empty();
-                if (data && data[0].popup_content) {
-                    $(data[0].popup_content).appendTo(self.$target.find(".o_popup_content_dev"));
-                }
-            });
+        this.$target.on('click', '.o_edit_popup', function (ev) {
+            ev.stopPropagation();
+            self.$target.find('.o_newsletter_modal').modal('show');
+            // Prevent to overlap overlay when open dialog
+            self.trigger_up('deactivate_snippet');
+            self.$target.trigger('content_changed');
         });
-    },
-});
-
-WysiwygMultizone.include({
-    events: _.extend({}, WysiwygMultizone.prototype.events, {
-        'click #edit_dialog': 'edit_dialog',
-        'click .o_popup_modal_content [data-dismiss="modal"]': 'close_dialog',
-    }),
-    save: function () {
-        var $target = $('#wrapwrap').find('#o_newsletter_popup');
-        if ($target && $target.length) {
-            this.close_dialog();
-            $('.o_popup_bounce_small').show();
-            if (!$target.find('.o_popup_content_dev').length) {
-                $target.find('.o_popup_modal_body').prepend($('<div class="o_popup_content_dev" data-oe-placeholder="' + _t("Type Here ...") + '"></div>'));
-            }
-            var content = $('#wrapwrap .o_popup_content_dev').html();
-            var newsletter_id = $target.parent().attr('data-list-id');
-            ajax.jsonRpc('/web/dataset/call', 'call', {
-                model: 'mail.mass_mailing.list',
-                method: 'write',
-                args: [
-                    parseInt(newsletter_id),
-                    {'popup_content':content},
-                    this.options.recordInfo.context,
-                ],
-            });
-        }
         return this._super.apply(this, arguments);
     },
-    destroy: function () {
-        this.close_dialog();
-        this._super();
+    /**
+     * @override
+     */
+    cleanForSave: function () {
+        this.$('.o_newsletter_modal').addClass('d-none');
```

Comment:
Why do we need the d-none class ? Should always use standard bootstrap to hide/show the modal.